sharepoint-server-farm-ha/azuredeploy.json

```diff
@@ -0,0 +1,1985 @@
+{
+	"$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"newStorageAccountName": {
```

Comment:
You are running all the VMs and their data disks out of one storage account. Is that sufficient for the IO?